internal/mode/update_mode.go

```diff
@@ -1,10 +1,10 @@
-package engine
+package mode
```

Comment:
yeahhh it used to live in `engine` but iirc that didn't work because it needed to be used in `containerupdate`, and then i tried to put it in `model` but that didn't work b/c it needs some `build` stuff...? I... think? let me look and see what I can do.

Reply:
oh right! `mode` needs `k8s` for [ProvideUpdateMode](https://github.com/windmilleng/tilt/blob/f0a4915e77040559b6da1e2acc6f340038b4dc5f/internal/mode/update_mode.go#L45).